### ISSUE TITLE
Integration with other packages

### ISSUE THREAD

**Issue by jlapeyre** [QuantumBFS/Yao.jl#157]
[QuantumGlue.jl](https://github.com/jlapeyre/QuantumGlue.jl) implements the most basic conversion between `DefaultRegister` and a [QuantumOptics.jl](https://github.com/qojulia/QuantumOptics.jl) type. I am doing this mostly as a way to get an overview of Julia's quantum package landscape.  I wonder how this fits with your plans. I don't want to duplicate effort.

**Comment by GiggleLiu** [comment]
I think the information “we are going to make a meta package before the end of this year” is important. We are happy to work out a reasonable abstraction together with QuantumGlue and QuantumOptics, this is quite important to building a constructive Julia ecosystem.

@Roger-luo , let’s start a new meta package so that we can hear advices from Quantum Julia guys better?

**Comment by jlapeyre** [comment]
Hi. Thanks for the comments. I did make a change or two following your suggestions. I read about the batches and active qubits, but did not see an easy way make them work with `Ket`.

> I wonder if it is possible to work out a reasonable abstraction together with QuantumGlue and QuantumOptics, this is quite important for a constructive Julia ecosystem, and maybe better than glue?

Yes, Julia quantum packages are somewhat fragmented. The glue package is exploratory. Using some common conventions or codes would be great. As usual, it's a matter of developer hours.

**Comment by Roger-luo** [comment]
Indeed, they are designed for different purpose. I would suggest to convert a `register` with batch size 1 (batch size is usually a type parameter, so it can be dispatched) to `ket` and convert a `register` with many batches to a vector of `ket`.
